docs/jade-getting-started.md

```diff
@@ -177,15 +179,16 @@ open -a "IntelliJ IDEA CE"
 then click in the search box and install **Cloud Code**, which integrates
 Google Cloud features with IntelliJ IDEA.
 
-## 5. Create GitHub token
+## 6. Create GitHub token
 
 The GitHub token verifies team permissions. This token is necessary for the next
 step, [Login to Vault](#6-login-to-vault). To create a token:
 
 1. Go to the [GitHub Personal Access Token](https://github.com/settings/tokens)
 page and click **Generate new token**.
-2. Give the token a descriptive name, **only** give it the `read:org` scope
-under `admin:org`, and click **Generate token**.
+2. Give the token a descriptive name, **only** give it the following two scopes and then click **Generate token**.
+  *  `read:org` scope under `admin:org`
+  * `workflow` (this will give you access to kick off github actions from the command line)
```

Comment:
@samanehsan  - I know you just ran into the error when you were trying to run ./test-dispatch.sh from the command line. Was this the solution for you?

Reply:
My token has workflow and full repo access, which is probably too permissive